test/functional/api/vim_spec.lua

```diff
@@ -1267,5 +1268,68 @@ describe('API', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
+  describe('statusline', function()
+    before_each(function()
+      screen = Screen.new(40, 4)
+      screen:attach()
+      command('set laststatus=2')
+		end)
```

Comment:
For me this has not the right indentation because of tabs.

Reply:
Sorry, dont know how that happend! Fixed it.